code/drasil-example/dblpend/lib/Drasil/DblPend/Body.hs

```diff
@@ -150,7 +150,7 @@ symbMap = cdb (map (^. output) iMods ++ symbolsAll) ideaDicts conceptChunks []
 
 -- | Holds all references and links used in the document.
 allRefs :: [Reference]
-allRefs = [externalLinkRef]
+allRefs = externalLinkRef : SRS.sectionReferences ++ map ref (labelledContent ++ funcReqsTables)
```

Comment:
Was this really needed? The `fillcdbSRS` function should be doing this already for any `LabelledContent` registered in the `ChunkDB`. (I'm not saying it should be doing that, only that it currently does that.)